source/Client/TextView.cpp

```diff
@@ -652,7 +652,7 @@ void CTextView::LoadCache(void)
 				DWORD dwLineSize;
 				if (m_dwNumCachedLines + 1 < dwNumLines)
 				{
-					const CLineInfo& rNextLineInfo = m_arrLines[(size_t)(m_dwNumCachedLines + 1)];
+					const CLineInfo& rNextLineInfo = m_arrLines[((size_t)m_dwNumCachedLines + 1)];
```

Comment:
For me, those parentheses are excessive. It looks nicer if we remove them:
```suggestion
					const CLineInfo& rNextLineInfo = m_arrLines[(size_t)m_dwNumCachedLines + 1];
```

But it is my opinion, which may be unrelevant to the project codestyle and @bchavez opinion.